.github/pull_request_template.md

```diff
@@ -1,2 +1,45 @@
-<!-- Please refer to the related issue number -->
-closes #
+## Description
+
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->
+
+Closes # (issue)
+
+## Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Code refactoring (Restructuring existing code w/o changing its external behavior)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Previews
+
+<!-- Please paste screenshots to reveal changes in this PR. -->
+
+## Checklist
+
+- [ ] My code follows the style guidelines of this project.
+- [ ] I have performed a self-review of my own code.
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] My changes generate no new warnings.
+- [ ] Any dependent changes have been merged and published in downstream modules.
+- [ ] I have checked my code and corrected any misspellings.
+- [ ] I've read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.
```

Comment:
Should we mark them as checked by default? That would be convenient for most of us.

Reply:
I'm afraid that contributors will ignore them if this list is checked by default.

Reply:
I think it's on-demand. If a pull request didn't touch any code, it can just leave those checkboxes to uncheck.

Reply:
OK.